pkg/graph/resolvers/plan_payments.go

```diff
@@ -0,0 +1,65 @@
+package resolvers
+
+import (
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/mint-app/pkg/models"
+	"github.com/cmsgov/mint-app/pkg/storage"
+)
+
+// PlanPaymentsCreate handles requests to create a Plan Payment
+func PlanPaymentsCreate(
+	logger *zap.Logger,
+	store *storage.Store,
+	payments *models.PlanPayments,
+) (*models.PlanPayments, error) {
+
+	return store.PlanPaymentsCreate(logger, payments)
+}
+
+// PlanPaymentsRead handles requests to find a Plan Payment by ID
+func PlanPaymentsRead(
+	logger *zap.Logger,
+	store *storage.Store,
+	id uuid.UUID,
+) (*models.PlanPayments, error) {
+
+	return store.PlanPaymentsRead(logger, id)
+}
+
+// PlanPaymentsReadByModelPlan handles requests to find a Plan Payment by model plan association
+func PlanPaymentsReadByModelPlan(
+	logger *zap.Logger,
+	store *storage.Store,
+	id uuid.UUID,
+) (*models.PlanPayments, error) {
+
+	return store.PlanPaymentsReadByModelPlan(logger, id)
+}
+
+// PlanPaymentsUpdate handles requests to update a Plan Payment
+func PlanPaymentsUpdate(
```

Comment:
@OddTomBrooks , do you have unit tests for this?

Reply:
Also, `scripts/dev test:go` is failing right now. I think this is because the table list used to truncate the db needs to have the newly added table added to it. That happens in 2 places
1. truncate.go ~ line 20 ( needs to be listed before model plan)
2. scripts/dev ~ line 381 ( also needs to be listed before model plan)